release notes/v0.53.0.md

```diff
@@ -0,0 +1,230 @@
+k6 `v0.53.0` is here 🎉! This release includes:
+
+- Native ECMAScript modules support :tada:!
+- New experimental OpenTelemetry metrics output :rocket:!
+- Blob support in experimental websockets module
+- Consolidating cloud features and commands under `k6 cloud` :cloud:
+- Breaking change: remove magic URL resolutions :stop_sign:
```

Comment:
I would prefer if we drop all the emojis, they don't add any additional value. IMHO.